source/languages/en/riak/community/product-advisories/appconfig-forward.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Forward Incompatibility of app.config
+project: riak
+version: 1.0.0+ 
+versions: false
+document: reference
+---
+
+Info | Value
+:----|:-----
+Date issued | April 15, 2015
+Product | Riak
+Affected Riak versions | 2.0.0+, 2.1.0+
+
+## Overview
+
+A forward incompatibility of app.config between 1.x and 2.x has been identified.
+
+## Description
+
+There is a forward compatibility issue when upgrading from the 1.x series to the 2.x series and continuing to use an app.config instead of migrating to the new riak.conf introduced in 2.0. This issue results in different default bucket properties being applied to un-typed buckets in 2.x, specifically for the `allow_mult` and `dvv_enabled` parameters. In 1.x `allow_mult` defaulted to 'false' but in 2.x instead defaults to 'true'. The `dvv_enabled` parameter did not exist in 1.x, and defaults to 'true' in 2.x.
```

Comment:
Should 'app.config' and 'riak.conf' be wrapped in backticks? I think I've seen some pages do that, but there are probably others that don't. 

Do we care enough to decide on a consistent style, and use that here + going forward?